CMakeLists.txt

```diff
@@ -266,15 +266,105 @@ add_library(faddeeva STATIC src/Faddeeva.c)
 #===============================================================================
 
 set(program "openmc")
-file(GLOB source src/*.F90 src/xml/openmc_fox.F90)
-add_executable(${program} ${source})
+set(LIBOPENMC_FORTRAN_SRC
```

Comment:
It's actually a recommendation to [not use `GLOB` to list source files](https://cmake.org/cmake/help/latest/command/file.html?highlight=glob#file).